### PR TITLE
Update Swift 2.3 branch

### PR DESCRIPTION
**Pull request by jessedc**
Update with master branch, plus couple of changes to get the framework and demo project running in Xcode 8 / Swift 2.3. Also updates travis

### CODE REVIEW

File: ActiveLabel.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		880C59C31D88E201000E9C18 /* ActiveLabel.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 8F0249A21B9989B1005D8035 /* ActiveLabel.framework */; };
```
Comment (poolqf) [inline ActiveLabel.xcodeproj/project.pbxproj]: Why this change? This is not in the master branch

File: ActiveLabel.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -69,6 +74,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				880C59C31D88E201000E9C18 /* ActiveLabel.framework in Frameworks */,
```
Comment (poolqf) [inline ActiveLabel.xcodeproj/project.pbxproj]: Same as https://github.com/optonaut/ActiveLabel.swift/pull/108/files#r78713377

File: ActiveLabel/ActiveLabel.swift [hunk 3]
```diff
@@ -395,19 +412,19 @@ typealias ElementTuple = (range: NSRange, element: ActiveElement, type: ActiveTy
...
     }
-    
-    public override func touchesCancelled(touches: Set<UITouch>, withEvent event: UIEvent?) {
-        guard let touch = touches.first else { return }
+
+    public override func touchesCancelled(touches: Set<UITouch>?, withEvent event: UIEvent?) {
+        guard let touch = touches?.first else { return }
         onTouch(touch)
-        super.touchesCancelled(touches, withEvent: event)
+        super.touchesCancelled(touches!, withEvent: event)
```
Comment (poolqf) [inline ActiveLabel/ActiveLabel.swift]: calling super should not require unwrapping touches since the parameter is optional

**Comment by poolqf** [comment]
Hi @jessedc , thanks for this. To be honest I totally forgot to update it with master.

Just minor comments, and this PR will be ready to merge 🎉

File: ActiveLabel/ActiveLabel.swift (hunk 3 above)
Comment (jessedc) [inline ActiveLabel/ActiveLabel.swift]: This was suggested by the migrator, upon inspection the `touches:` parameter is not optional. (` touchesCancelled(touches: Set<UITouch>, withEvent event: UIEvent?)`

File: ActiveLabel.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jessedc) [inline ActiveLabel.xcodeproj/project.pbxproj]: This includes the ActiveLabel.framework as a dependency of ActiveLabelDemo target. I was unable to build the demo target without declaring the framework dependency.

**Comment by jessedc** [comment]
I'd love the travis build to be passing at this point, but it looks like it's an issue with xctool?

**Comment by yoshiboarder** [comment]
xcode 8 is not working when I use the storyboard

**Comment by poolqf** [comment]
Hi @jessedc , sorry for the delay. Thank you for keeping `swift-2.3` up to date.

Just merged your PR and fixed a couple of issues in https://github.com/optonaut/ActiveLabel.swift/commit/fcc52b30c885b771940fba99af0234a6747be905
